packages/core/src/controllers/AccountController.ts

```diff
@@ -37,9 +36,17 @@ export interface AccountControllerState {
   preferredAccountType?: W3mFrameTypes.AccountType
   socialWindow?: Window
   farcasterUrl?: string
+  is1ClickAuthenticating?: boolean
   provider?: UniversalProvider | Provider | CombinedProvider
   status?: 'reconnecting' | 'connected' | 'disconnected' | 'connecting'
-  siweStatus?: 'uninitialized' | 'ready' | 'loading' | 'success' | 'rejected' | 'error'
+  siweStatus?:
```

Comment:
can't you use `SiweStatus` ?

Reply:
Nope because you'd have to import the SIWE package